docs/src/components/copy-button.tsx

```diff
@@ -0,0 +1,52 @@
+import * as React from "react";
+import { CheckIcon, CopyIcon } from "@radix-ui/react-icons";
+
+import { cn } from "@/lib/utils";
+import { Button } from "@/components/ui/button";
+
+interface CopyButtonProps extends React.HTMLAttributes<HTMLButtonElement> {
+  value: string;
+  src?: string;
+}
+
+async function copyToClipboard(value: string) {
+  navigator.clipboard.writeText(value);
+}
+
+export function CopyButton({
+  value,
+  className,
```

Comment:
You never send a className to this component; why add the additional complexity of supporting it prematurely?